tools/tester.rb

```diff
@@ -19,15 +20,21 @@
   end
 end
 
+$ARGS.delete_if do |arg|
+  if (m = /\A\-bc=(standard|cannon)/.match(arg))
+    $filter_run = m[1]
```

Comment:
Can we name this `filter_config` or something? Do we even need this flag?

Reply:
We do not need it, if we keep the `// cannon` modifier inside the test cases. I added this to allow running all tests with a specific baseline compiler without modifiying each file. I will remove it.